torch/sparse/__init__.py

```diff
@@ -137,3 +139,55 @@ def sum(input, dim=None, dtype=None):
             return torch._sparse_sum(input, dim, dtype=dtype)
         else:
             return torch._sparse_sum(input, dtype=dtype)
+
+
+def softmax(input, dim, dtype=None):
+    # type: (Tensor, int, Optional[int]) -> Tensor
```

Comment:
we python 3 only, can do these annotations inline!

Reply:
Fixed.